### PR TITLE
Base doc auth service

### PR DESCRIPTION
**Pull request by AyWa**
Just base doc we need to discuss about what we want to put inside.
- [ ] I guess grammar / spell check needed.
- [x] SAML basic
- [x] LDAP basic
- [x] CAS vasic
- [x] db login basic
- [x] jwt basic
- [x] oauth2 basic

what do we want to put ?

### CODE REVIEW

File: docs/guide/enterprise/login_support.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Login support
+
+In enterprise environments, Zepl support multiple login systems. This is the list of all the integrations we support, but be free to contact us if you need or have any questions about an integration.
+
+## DB
+Zepl default login system. The user will be authentificate against our own Database.
```
Comment (minahlee) [inline docs/guide/enterprise/login_support.md]: `authentificate` -> `authenticated`?

File: docs/guide/enterprise/login_support.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+#### Register flow
+- user register with email adress in zepl login page.
+- user receive an email with a link
+- user click on the link an create an account (set username and password)
```
Comment (minahlee) [inline docs/guide/enterprise/login_support.md]: `an create an account` -> `and create an account`

**Review by minahlee — COMMENTED** [review]
It will be good to have convention to use either `ZEPL` or `Zepl` or `zepl` for consistency.

Also would be nice to mention at the beginning that we are handling this integration when we install.

**Comment by AyWa** [comment]
@minahlee I updated, please let me know if I can improve something

**Review by minahlee — APPROVED** [review]

**Review by sejunra — APPROVED** [review]
made direct changes to the doc.

**Pull request merged**